README.md

```diff
@@ -1,18 +1,18 @@
 # ReVanced Extended Builder
 
-This project will allow you to download the APK of [supported](https://github.com/inotia00/revanced-patches/tree/revanced-extended#-patches) apps and build ReVanced Extended easily!
+This project will allow you to download the APK of [supported](https://github.com/inotia00/revanced-patches?tab=readme-ov-file#-list-of-patches-in-this-repository) apps and build ReVanced Extended easily!
 
 ## Required
 
-You'll need at least [Zulu JDK 17](https://www.azul.com/downloads/?version=java-17-lts&package=jdk) and [ADB](https://developer.android.com/studio/command-line/adb) (optional, required only for rooted phones).
+You'll need at least [Zulu JDK 17](https://www.azul.com/downloads/?version=java-17-lts&package=jdk).
```

Comment:
The docs doesn't mention this requirement. Is Zulu JDK 17 installed automatically when rvx-builder is ran?
Should this line be removed from the readme to avoid confusion?

Reply:
If JAVA is not installed, the builder will automatically download JAVA (portable)

Yeah, this is optional

You can remove this instruction